src/app/components/bitcoin-contract-entry-point/bitcoin-contract-entry-point-layout.tsx

```diff
@@ -30,21 +28,23 @@ export function BitcoinContractEntryPointLayout(props: BitcoinContractEntryPoint
 
   return (
     <Flex cursor={cursor} onClick={onClick} outline={0}>
-      <Flag img={icon} align="middle" spacing="base" width="100%">
+      <Flag align="middle" img={icon} spacing="space.04" width="100%">
         <HStack alignItems="center" justifyContent="space-between" width="100%">
-          <Text>{'Bitcoin Contracts'}</Text>
+          <styled.span textStyle="label.01">Bitcoin Contracts</styled.span>
           <Tooltip
             label={formattedBalance.isAbbreviated ? balance.amount.toString() : undefined}
             placement="left-start"
           >
-            <Text fontVariantNumeric="tabular-nums" textAlign="right">
+            <styled.span textStyle="label.01">
               {isLoading ? <LoadingSpinner size="sm" /> : formattedBalance.value}
-            </Text>
+            </styled.span>
           </Tooltip>
         </HStack>
-        <HStack height="1.25rem" alignItems="center" justifyContent="space-between" width="100%">
-          <Caption>{caption}</Caption>
-          <Flex>{isLoading ? '' : <Caption>{usdBalance}</Caption>}</Flex>
+        <HStack alignItems="center" justifyContent="space-between" width="100%">
+          <styled.span textStyle="caption.02">{caption}</styled.span>
+          <Flex>
+            {isLoading ? '' : <styled.span textStyle="caption.02">{usdBalance}</styled.span>}
```

Comment:
Maybe we should have a `LoadingSpinner` here too when `isLoading` ?

Reply:
Yeah, possibly, this is code implemented by the DLC.Link team, but I can try to check it.